src/views/well-architected-framework/index.tsx

```diff
@@ -0,0 +1,61 @@
+import slugify from 'slugify'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import { generateTopLevelSidebarNavData } from 'components/sidebar/helpers'
+import ProductViewContent from 'views/product-tutorials-view/components/product-view-content'
+import HeroHeadingVisual from 'views/product-landing/components/hero-heading-visual'
+import { SidebarProps } from 'components/sidebar'
+import OverviewCta from 'views/product-landing/components/overview-cta'
+import { WellArchitectedFrameworkLandingProps } from './types'
+import s from './well-architected-framework-landing.module.css'
+import { EnrichedLinkNavItem } from 'components/sidebar/types'
+
+export const wafData = {
+	slug: 'well-architected-framework',
+	name: 'Well Architected Framework',
+}
+
+export default function WellArchitectedFrameworkLandingView(
+	props: WellArchitectedFrameworkLandingProps
+) {
+	const { data, layoutProps, metadata } = props
+	const { hero, overview, blocks } = data.pageData
+
+	return (
+		<SidebarSidecarLayout
+			headings={layoutProps.headings}
+			breadcrumbLinks={layoutProps.breadcrumbLinks}
+			sidebarNavDataLevels={[
+				generateTopLevelSidebarNavData(metadata.name) as SidebarProps,
+				{
+					title: metadata.name,
+					levelButtonProps: {
+						levelUpButtonText: 'Main Menu',
+						levelDownButtonText: 'Previous',
+					},
+					menuItems: [
+						{
+							title: 'Overview',
+							isActive: true,
+							fullPath: `/${metadata.slug}`,
+							id: metadata.slug,
+						} as EnrichedLinkNavItem,
+						...layoutProps.sidebarSections,
+					],
+					showFilterInput: false,
+				},
+			]}
```

Comment:
You should be able to set the `"Overview"` item setting the `overviewItemHref` property that is sent to `Sidebar`. That prop handles consistently rendering this item. 😊

I'd like to see this updated, but totally feel free to do it on a follow-up PR!

```suggestion
			sidebarNavDataLevels={[
				generateTopLevelSidebarNavData(metadata.name) as SidebarProps,
				{
					title: metadata.name,
					levelButtonProps: {
						levelUpButtonText: 'Main Menu',
						levelDownButtonText: 'Previous',
					},
					overviewItemHref: `/${metadata.slug}`,
					menuItems: layoutProps.sidebarSections,
					showFilterInput: false,
				},
			]}
```

Reply:
Also will openly admit that the TypeScript around `sidebarNavDataLevels` is not great! And falls short at exposing things like `overviewItemHref` if you aren't already aware of the prop.

Reply:
Ah okay! ty for this tip 😅  I'll check that out